react-app/src/Home.js

```diff
@@ -1,21 +1,154 @@
 import { StatusBar } from 'expo-status-bar'
-import { StyleSheet, View, Button } from 'react-native'
+import { StyleSheet, View, FlatList, Text, Pressable } from 'react-native'
 import NavBar from './NavBar.js'
 import React from 'react'
+import { DUMMY_DATA } from '../data/DummyData'
+import { SvgXml } from 'react-native-svg'
+import ClockIcon from '../assets/icons/clock.svg'
+import ArrowIcon from '../assets/icons/arrow-right.svg'
 
-export default function Home({ navigation }) {
+// Component of the card
```

Comment:
These comments doesn't describe more that the name of the function. Consider removing.